### ISSUE TITLE
Password Recovery 

### ISSUE THREAD

**Issue by FernandoJRR** [noisrucer/girok#42]
## Description
I think it's a good idea to make available a change password, I haven't used Girok in a while since I forgot my password and I cannot create a new account nor change it.

## Purpose
Make easy to recover lost passwords

**Comment by noisrucer** [comment]
I agree with you. I'll work on reset password soon.

Btw, did you have many data for your account?

**Comment by FernandoJRR** [comment]
After trying for some time I got the correct password and entered my account. Still, password recovery would be great tnx.

**Comment by HajunKim** [comment]
I agree with Fernado. password recovery is indeed essential. Regardless, I have great admiration for this project.
